### DISCUSSION TITLE
Reduce the memory overhead and CPU usage on web

### DISCUSSION THREAD

**Discussion by AbdalaMask** [unoplatform/uno#9714] [Q&A]
how fix to bug
Reduce the memory overhead and CPU usage on web
![SWQUJJ}_% O1OY%IMH%QYIR](https://user-images.githubusercontent.com/37013638/188104824-4279a3e1-63fd-4b8f-9d38-7df04f810e00.png)

**Comment by jeromelaban** [comment]
Thanks for the report. Unfortunately, there's nothing specific enough to help you. If you have a specific repro case, let us know.
